commit 12698ab87b6d11b04f59da4796f3e8c770989031
Author: 悦咚 / Dark495 <flandrestudio.cn@gmail.com>
Date:   Sat Apr 26 21:54:20 2025 +0800

    feat: seek submit list

diff --git a/src/page/user/_component/UserTorrentSeekSubmitListView.tsx b/src/page/user/_component/UserTorrentSeekSubmitListView.tsx
new file mode 100644
index 00000000..6b328882
--- /dev/null
+++ b/src/page/user/_component/UserTorrentSeekSubmitListView.tsx
@@ -0,0 +1,10 @@
+import { connect } from "react-redux";
+import { withTranslation } from "react-i18next";
+import { UserTorrentListBaseView, mapAction } from "./UserTorrentListBaseView";
+import {UserTorrentSeekListView} from "@app/page/user/_component/UserTorrentSeekListView";
+
+class UserTorrentSeekSubmitListView extends UserTorrentSeekListView {
+  protected type = "SEEK_SUBMIT";
+}
+
+export default connect(null, mapAction)(withTranslation()(UserTorrentSeekSubmitListView));
