commit 00448ccbf51954472cd47e9142ba42a8ee46be4d
Author: 悦咚 / Dark495 <flandrestudio.cn@gmail.com>
Date:   Sat Apr 26 18:18:21 2025 +0800

    feat master: editor paste image

diff --git a/src/app.env.ts b/src/app.env.ts
index 68f44de8..5ed43fbe 100644
--- a/src/app.env.ts
+++ b/src/app.env.ts
@@ -9 +9,4 @@ const secret = _.join(["H", "L", "k", "P", "c", "W", "m", "y", "c", "L", "5", "7
-
+const imgApi = {
+  url: "https://img.m-team.cc/api/1/upload",
+  key: "chv_R_69c9b8590fa411662aecfa580ef6553fedafce215c569b7cadb0fca08e4966b9a7b29272798af37a7d1e45a8984d223179bbc926d814c4fe1c1d2753f362b1b0",
+};
@@ -32 +35,2 @@ const Server = {
-    secret
+    secret,
+    imgApi,
